README.md

```diff
@@ -106,397 +113,589 @@ Please read the information above to understand the table
 
 ```bash
 open-attestation encrypt ./examples/wrapped-documents/example.0.json ./tmp/encrypted.json
+```
 
+The response looks like:
+```
 ✔  success   Encrypted document saved to: tmp/encrypted.json
 ⚠  warning   Here is the key to decrypt the document: don't lose it: 9bac5be27bac31d852fc1e48eb9d5249ec6ad7978da23377b5879f7a24994cb2
 ```
 
-### Decrypting document
+### Decrypting a document
 
-This allows you to decrypt document encrypted using the method above.
+Using the `decrypt` command, you can decrypt documents that were encrypted using the [encrypt](#encrypting-a-document) method.
+
+The following shows the syntax of the `decrypt` command:
 
 ```bash
 open-attestation decrypt <input> <output> <key>
 ```
 
-Example:
+The following is a command example:
 
 ```bash
 open-attestation decrypt ./src/__tests__/fixture/did-dns-encrypted.json decrypted.json 88da9b9cd61cfc1677ae7e79dba9b3aeba4b40c95f94c950759e76c6210b5402
+```
+The response looks like:
 
+```
 ✔  success   Decrypted document saved to: decrypted.json
 ```
 
 ### Token Registry
 
-#### Deploy new Token Registry
+#### Deploying a new Token Registry
 
-Deploys a token registry contract on the blockchain. Factory Contract that have been deployed using token-registry can be used with the factory address flag. To deploy a standalone token registry, please refer to [Token-Registry](https://github.com/Open-Attestation/token-registry) deployment.
+The `deploy token-registry` command deploys a token registry contract on the blockchain. You can use the `--factory-address` flag with the Factory Contract that were deployed using this command. 
+
+To deploy a standalone token registry, refer to the [Token-Registry](https://github.com/Open-Attestation/token-registry) repository.
+
+The following shows the syntax of the `deploy token-registry` command:
 
 ```bash
 open-attestation deploy token-registry <registry-name> <registry-symbol> --factory-address <factory-address> [options]
 ```
+See more options [here](#providing-the-wallet).
 
-Example - with private key set in `OA_PRIVATE_KEY` environment variable (recommended). [More options](#providing-the-wallet).
+The following command is a recommended example with the private key set in the `OA_PRIVATE_KEY` environment variable. 
 
 ```bash
 open-attestation deploy token-registry "My Sample Token" MST --network sepolia
+```
+
+The response looks like:
 
+```
 ✔  success   Token registry deployed at 0x4B127b8d5e53872d403ce43414afeb1db67B1842
 ```
 
-#### Issue document to token registry
+#### Issuing a document to token registry
+
+The `token-registry issue` command issues a hash to a token registry deployed on the blockchain. The `--tokenId` option indicates the document hash, and the `--to` option indicates the title escrow address to which the document is mapped.
+
+<!--Flag: The syntax below didn't mention the `--to` option. The command example mentions `--to` option, but did not show how to use `--beneficiary` or `--holder` options.-->
```

Comment:
Thanks for pointing this out! Let me check on it and get back to you

Reply:
updated the command & response

Reply:
thanks Chloe! Added some comments on this